apps/python/scalert.py

```diff
@@ -123,6 +125,21 @@ def init(self):
         except:
             pass
 
+        try:
+            self._phaseNumber = self.configGetInt("constraints.phaseNumber")
+        except:
+            self._phaseNumber = 1
```

Comment:
set the default in the constructor only and use pass here. It might be confusing otherwise with the constructor.